test/functional/tools/creates_html.xml

```diff
@@ -0,0 +1,14 @@
+<tool id="creates_html" name="creates_html" version="1.0.0">
+  <command>
+    mkdir '$output.extra_files_path';
```

Comment:
&& should be used to concatenate commands, which also requires the use of CDATA.